web/quiz_views.py

```
@@ -569,21 +779,43 @@
+            answer_data = answers[q_id]
+            user_answer_obj = {
+                "question": question,
+                "user_answer": answer_data.get("user_answer", ""),
+                "is_correct": answer_data.get("is_correct", False),
+                "is_graded": answer_data.get("is_graded", False),
+                "points": answer_data.get("points_awarded", 0) if answer_data.get("is_graded", False) else (question.points if answer_data.get("is_correct", False) else 0),
+                "feedback": answer_data.get("feedback", ""),
+            }
+            user_answers.append(user_answer_obj)
+
+    for id, data in answers.items():
```

Ruff (0.8.2)

Loop control variable `data` not used within loop body

Rename unused `data` to `_data`

(B007)